### DISCUSSION TITLE
Maximum number of auto-add workflows: Individual / Pro / Team ?

### DISCUSSION THREAD

**Discussion by planetf1** [community/community#137767] [Projects and Issues] [answered]
### Select Topic Area

Question

### Feature Area

Projects

### Body

I am a free github user, and have a personal project board.

Using workflows I can add an auto-add workflow so that issues from a selected repo will automatically be added when meeting an appropriate search criteria

However the limit seems to be 1 workflow. When I try to add a second I get:

Duplicate workflow
Maximum number of auto-add workflows reached. Delete a workflow or upgrade your plan to continue.

How many are allowed on Pro or Team? I did find one old reference to 5, but this seems a surprisingly low number? Is that still the case?

In this case I am evaluating engagement in some repos, so want to overlay some categorizations on those issues without (yet) involving the community. That will follow. The number of repos is ~5-10. A project board is a good fit for this

This also means adding actions to the repos that are the source of the issues is out for now - though more viable after an initial evaluation.
An alternative is simply (...) custom code that uses the github API.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by planetf1** [comment, marked as answer]
https://docs.github.com/en/issues/planning-and-tracking-with-projects/automating-your-project/adding-items-automatically has the answer

For pro & team, it's 5 - so very limited

**Reply by sjsadowski** [reply]
Even worse than that, the number of "Default Workflows" exceeds 5, and can't be changed or removed, so the effective answer is 0 if your needs are different than GitHub's
